codegen/src/main/scala/akka/grpc/gen/javadsl/play/PlayJavaClientCodeGenerator.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.gen.javadsl.play
+
+import akka.grpc.gen.javadsl.{ JavaCodeGenerator, Service }
+import com.google.protobuf.compiler.PluginProtos.CodeGeneratorResponse
+import templates.PlayJava.txt.{ AkkaGrpcClientModule, ClientProvider }
+
+import scala.annotation.tailrec
+
+object PlayJavaClientCodeGenerator extends JavaCodeGenerator {
+  override def name: String = "akka-grpc-play-client-java"
+
+  override def perServiceContent = super.perServiceContent + generateClientProvider
+
+  private val generateClientProvider: Service => CodeGeneratorResponse.File = service => {
+    val b = CodeGeneratorResponse.File.newBuilder()
+    b.setContent(ClientProvider(service).body)
+    b.setName(s"${service.packageName.replace('.', '/')}/${service.name}ClientProvider.java")
+    b.build
+  }
+
+  override def staticContent(allServices: Seq[Service]): Set[CodeGeneratorResponse.File] = {
+    if (allServices.nonEmpty) {
+      val packageName = packageForSharedModuleFile(allServices)
+      val b = CodeGeneratorResponse.File.newBuilder()
+      b.setContent(AkkaGrpcClientModule(packageName, allServices).body)
+      b.setName(s"${packageName.replace('.', '/')}/AkkaGrpcClientModule.java")
+      val set = Set(b.build)
+      println(s"Generated [${packageName}.AkkaGrpcClientModule] add it to play.modules.enabled and a section " +
```

Comment:
extract this name as val AkkaGrpcClientModule, it's used in more places, in case we rename it